Movie/Movie/DetailModule/View/DescriptionViewController.swift

```diff
@@ -0,0 +1,258 @@
+            ratingLabel.topAnchor.constraint(equalTo: movieNameLabel.bottomAnchor, constant: 15),
+            ratingLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor, constant: 0),
+            genreLabel.topAnchor.constraint(equalTo: ratingLabel.bottomAnchor, constant: 5),
+            genreLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor, constant: 0),
+            genreLabel.widthAnchor.constraint(equalToConstant: 300),
+            seeMovieButton.topAnchor.constraint(equalTo: genreLabel.bottomAnchor, constant: 10),
+            seeMovieButton.centerXAnchor.constraint(equalTo: view.centerXAnchor, constant: 0),
+            seeMovieButton.heightAnchor.constraint(equalToConstant: 70),
+            seeMovieButton.widthAnchor.constraint(equalToConstant: 250),
+            actorsLabel.topAnchor.constraint(equalTo: seeMovieButton.bottomAnchor, constant: 20),
+            actorsLabel.leadingAnchor.constraint(equalTo: backgroundBlackView.leadingAnchor, constant: 20),
+            actorCollectionView.topAnchor.constraint(equalTo: seeMovieButton.bottomAnchor, constant: 50),
+            actorCollectionView.leadingAnchor.constraint(equalTo: backgroundBlackView.leadingAnchor, constant: 0),
+            actorCollectionView.trailingAnchor.constraint(equalTo: backgroundBlackView.trailingAnchor, constant: 0),
+            actorCollectionView.heightAnchor.constraint(equalToConstant: 250),
+            actorCollectionView.widthAnchor.constraint(equalToConstant: backgroundBlackView.frame.width),
+            descriptionTitleLabel.topAnchor.constraint(equalTo: actorCollectionView.bottomAnchor, constant: 15),
+            descriptionTitleLabel.leadingAnchor.constraint(equalTo: backgroundBlackView.leadingAnchor, constant: 20),
+            descriptionLabel.topAnchor.constraint(equalTo: descriptionTitleLabel.bottomAnchor, constant: 10),
+            descriptionLabel.leadingAnchor.constraint(equalTo: backgroundBlackView.leadingAnchor, constant: 20),
+            descriptionLabel.widthAnchor.constraint(equalToConstant: 360)
+        ])
+    }
+}
+
+// MARK: - UICollectionViewDataSource, UICollectionViewDelegate
+
+extension DescriptionViewController: UICollectionViewDataSource, UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        presenter?.actors.count ?? 0
+    }
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        cellForItemAt indexPath: IndexPath
+    ) -> UICollectionViewCell {
+        guard let cell = collectionView
+            .dequeueReusableCell(
+                withReuseIdentifier: Constants.actorCellIdentifier,
+                for: indexPath
+            ) as? ActorCollectionViewCell
+        else { return UICollectionViewCell() }
+        guard let actor = presenter?.actors[indexPath.row] else { return UICollectionViewCell() }
+        cell.configureCell(actor, networkService: networkManager)
```

Comment:
```suggestion
        cell.configureCell(actor, networkService:)
```
достать из презентера